dhis-2/dhis-web-embedded-jetty/pom.xml

```diff
@@ -89,10 +89,6 @@
       <artifactId>log4j-slf4j-impl</artifactId>
       <scope>compile</scope>
     </dependency>
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-orm</artifactId>
-    </dependency>
```

Comment:
Both of the removed deps in this pom were already removed in latest master #13710  & 2.40 #13757
Not sure why they weren't back ported. Maven build fails because these declared deps are not used.